frontend/src/Athlete/AthleteDisplay.jsx

```diff
@@ -78,18 +88,23 @@ const AthleteDisplay = () => {
     lastCreatedAthleteId.current = null;
   };
 
-/*   const handleEditClick = () => {
-    console.log("Edit ...");
-  };*/
+  const handleEditClick = (row) => {
+    if (searchBarRef.current) {
+      searchBarRef.current.clearSearch();
+    }
+    setSearchPar("");
+    setAthleteToEditId(athleteData[row].id);
+    setIsFormOpen(true);
+  };
 
   const actions = [
-/*     {
+    {
       name: "Edit",
       icon: <EditIcon />,
       onClick: handleEditClick,
       tip: "Edit basic information",
```

Comment:
To follow the same format, we can use uppercase letters for the first letters of each word